deployment/ansible/roles/bootstrap/molecule/default/molecule.yml

```diff
@@ -35,6 +35,8 @@ provisioner:
     group_vars:
       all:
         chain_id: test-net
+        company_name: CSA
```

Comment:
Do we have instructions to replace these values with custom ones for other companies?